libwaysip/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "libwaysip"
+description = "Library for facilitating wayland native area selection"
+version.workspace = true
```

Comment:
This keyword means to use the the version top workspace project. Before I set the workspace to libwaysip, but now you set nothing,so here you will get empty